extensions/amp-stream-gallery/1.0/README.md

```diff
@@ -0,0 +1,438 @@
+---
+$category@: layout
+formats:
+  - websites
+teaser:
+  text: Displays multiple similar pieces of content at a time, for features like related products or articles.
+experimental: true
+bento: true
+---
+
+# Bento Stream Gallery
+
+## Usage
+
+The Bento Stream Gallery is for displaying multiple similar pieces of content at a time along a horizontal axis. To implement a more customized UX, see
+[`amp-base-carousel`](https://amp.dev/documentation/components/amp-base-carousel/).
+
+_Example:_
+
+In the following example, we use `bento-stream-gallery` to display its content, showing a varying amount of its children at a time according to the viewport size.
+
+```html
+<bento-stream-gallery>
+  <img src="img1.png">
+  <img src="img2.png">
+  <img src="img3.png">
+  <img src="img4.png">
+  <img src="img5.png">
+  <img src="img6.png">
+  <img src="img7.png">
+</bento-stream-gallery>
+```
+
+Use Bento Stream Gallery as a web component ([`<bento-stream-gallery>`](#web-component)), or a Preact/React functional component ([`<BentoStreamGallery>`](#preact/react-component)).
+
+### Web Component
+
+You must include each Bento component's required CSS library to guarantee proper loading and before adding custom styles. Or use the light-weight pre-upgrade styles available inline. See [Layout and style](#layout-and-style).
+
+The examples below demonstrate use of the `<bento-stream-gallery>` web component.
+
+#### Example: Import via npm
+
+[example preview="top-frame" playground="false"]
+
+Install via npm:
+
+```sh
+npm install @ampproject/bento-stream-gallery
+```
+
+```javascript
+import '@ampproject/bento-stream-gallery';
+```
+
+[/example]
+
+#### Example: Include via `<script>`
+
+The example below contains an `bento-stream-gallery` with three sections. The
+`expanded` attribute on the third section expands it on page load.
+
+[example preview="top-frame" playground="false"]
+
+```html
+<head>
+  <script src="https://cdn.ampproject.org/custom-elements-polyfill.js"></script>
+  <script async src="https://cdn.ampproject.org/v0/bento-stream-gallery-1.0.js"></script>
```

Comment:
does this need `<style data-bento-boilerplate>` like the others?

Reply:
we're demo-ing the boilerplate inline styles on line 160, do I need to include it in this example as well?

Reply:
one of the two styles should be included here, feel free to use the `<link>` instead for brevity

Reply:
updated and fixed the other READMEs that didn't missed it as well